src/main/java/com/google/step/servlets/UserServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.step.servlets;
+
+import com.google.step.datamanager.UserManager;
+import com.google.step.datamanager.UserManagerDatastore;
+import com.google.step.model.Deal;
+import com.google.step.model.Restaurant;
+import com.google.step.model.Tag;
+import com.google.step.model.User;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles users. */
+@WebServlet("/api/users/*")
+public class UserServlet extends HttpServlet {
+
+  private UserManager userManager = new UserManagerDatastore();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
```

Comment:
Can you move it down closer to where the actual response json object is written?